### DISCUSSION TITLE
learning-golang/docs/package/comparison/

### DISCUSSION THREAD

**Discussion by giscus** [zhaohuabing/learning-golang#6] [Announcements]
# learning-golang/docs/package/comparison/

Go语言中的package和java类似，都用于将一组相关代码组织在一起，但两个语言对packag的使用又有不同之处。本文对比go和java package的异同，以帮助java程序员更好理解go的package。
Package声明
Java package声明
Java文件中的package声明 …

https://www.zhaohuabing.com/learning-golang/docs/package/comparison/

**Comment by zou-weidong** [comment]
good

**Comment by zou-weidong** [comment]
<script>alert(1)</script>
